src/Compiler/Driver/GraphChecking/FileContentMapping.fs

```diff
@@ -320,14 +339,20 @@ let (|NameofExpr|_|) (e: SynExpr) =
     match e with
     | SynExpr.App(flag = ExprAtomicFlag.NonAtomic; isInfix = false; funcExpr = SynExpr.Ident NameofIdent; argExpr = moduleNameExpr) ->
         match stripParen moduleNameExpr with
-        | SynExpr.Ident moduleNameIdent -> Some moduleNameIdent
+        | SynExpr.Ident moduleNameIdent -> Some(NameofResult.SingleIdent moduleNameIdent)
+        | SynExpr.LongIdent(longDotId = longIdent) ->
+            match longIdent.LongIdent with
+            | [] -> None
+            // This is highly unlikely to be produced by the parser
```

Comment:
Like when? :)

Reply:
I think this might be the case for custom operators. This will of course in practice never be a valid module name but theoretically speaking...